src/panels/developer-tools/assist/developer-tools-assist.ts

```diff
@@ -182,9 +192,19 @@ class HaPanelDevAssist extends SubscribeMixin(LitElement) {
     `;
   }
 
+  private _download() {
+    fileDownload(
+      `data:text/plain;charset=utf-8,${encodeURIComponent(
+        JSON.stringify({ results: this._results }, null, 2)
+      )}`,
+      `intent_results.json`
+    );
+  }
+
   static get styles(): CSSResultGroup {
     return [
       haStyle,
+      buttonLinkStyle,
```

Comment:
Please don't perpetuate use of this style if at all possible, and instead allow buttons to look like buttons.  I described the a11y issues in https://github.com/home-assistant/frontend/pull/14989#discussion_r1062723837.

Reply:
We could use the same approach as we do with Logs. Or move the button to the button bar.

Reply:
Used same approach as logs. We can't access the toolbar in dev tools because of the current architecture.

Updated screenshot:

![CleanShot 2023-07-11 at 14 38 47](https://github.com/home-assistant/frontend/assets/1444314/3e0df30f-ed71-4dba-9853-ea93d1d40bd1)

Reply:
Thanks 👍🏻